### PR TITLE
Add Module field and Update Commands

### LINKED ISSUE #51: Add class field to person

### PR DESCRIPTION
**Pull request by yongqqqq**
#### Main Changes:
- ```Module``` field has been added to ```Person``` class.
- Updated ```AddCommand``` and ```EditCommand``` to handle ```Module``` field.
- Updated ```Parser``` class to handle ```Module``` field for both command types.
- Modified test cases to ensure correct handling with the new ```Module``` field.

#### Additional Changes:
- Refactoring to ensure code consistency
- Fix codestyle issues

#### Testing:
- All existing and new test cases pass
- This verifies that the module field is being parsed and handled correctly.

closes #51

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -162,69 +179,77 @@ public void parse_compulsoryFieldMissing_failure() {
...
                 expectedMessage);
 
         // missing address prefix
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + GENDER_DESC_BOB
-                        + VALID_ADDRESS_BOB,
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + GENDER_DESC_BOB + VALID_ADDRESS_BOB + MODULE_DESC_BOB,
                 expectedMessage);
 
-        // missing gender prefix
```
Comment (Jianwen0451) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:184]: I think this part should be kept to test the gender prefix

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 2]
```diff
@@ -162,69 +179,77 @@ public void parse_compulsoryFieldMissing_failure() {
...
+        // missing module prefix
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                        + GENDER_DESC_BOB + ADDRESS_DESC_BOB + VALID_MODULE_BOB,
                 expectedMessage);
 
         // all prefixes missing
-        assertParseFailure(parser, VALID_NAME_BOB + VALID_PHONE_BOB + VALID_EMAIL_BOB + VALID_GENDER_BOB
-                        + VALID_ADDRESS_BOB,
+        assertParseFailure(parser, VALID_NAME_BOB + VALID_PHONE_BOB + VALID_EMAIL_BOB
+                        + GENDER_DESC_BOB + VALID_ADDRESS_BOB + VALID_MODULE_BOB,
```
Comment (Jianwen0451) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:208]: Here should be valid gender Bob

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 3]
```diff
@@ -162,69 +179,77 @@ public void parse_compulsoryFieldMissing_failure() {
...
 
-        // invalid tag
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + GENDER_DESC_BOB
-                + ADDRESS_DESC_BOB + INVALID_TAG_DESC + VALID_TAG_FRIEND, Tag.MESSAGE_CONSTRAINTS);
+        // invalid module
+        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
+                + GENDER_DESC_BOB + ADDRESS_DESC_BOB
+                + INVALID_MODULE_DESC + TAG_DESC_FRIEND, Module.MESSAGE_CONSTRAINTS);
 
-        // invalid gender
```
Comment (Jianwen0451) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:217]: Here should also be kept as well I think

**Review by RadieonAjax — CHANGES_REQUESTED** [review]
Looks good to me! There's one section that I have some questions about, but all's good otherwise.

**Review by RadieonAjax — APPROVED** [review]

**Pull request merged**